features/mogelijkOnjuist.feature

```diff
@@ -180,7 +180,7 @@ Functionaliteit: Mogelijk onjuist
       Dan bevat het antwoord property mogelijkOnjuist.huisnummer met de waarde true
       En bevat het antwoord property mogelijkOnjuist.huisletter met de waarde true
       En bevat het antwoord geen property mogelijkOnjuist.postcode
-      En bevat het antwoord property postcode met een waarde
+      En bevat het antwoord geen property postcode
       En bevat het antwoord property huisnummer met een waarde
       En bevat het antwoord geen property huisletter
```

Comment:
Als gegeven huisletter een waarde heeft, dan wordt deze altijd geleverd i.c.m. huisnummer, ook als er met de fields parameter alleen wordt gevraagd om het huisnummer.
Nu is het hier niet duidelijk of huisletter een waarde heeft, er kan nl. ook een onderzoek naar huisletter gestart zijn omdat deze een waarde had moeten hebben, maar niet heeft in dat geval leveren we de huisletter niet, omdat die geen waarde heeft.
Afhankelijk van of huisletter een waarde heeft of niet wordt deze hier dus wel of niet geleverd.

Reply:
die precognitie ontbrak inderdaad in het scenario. Bij Gegeven toegevoegd:
      En hebben gegevens postcode en huisnummer een waarde
      En hebben gegevens huisletter en huisnummertoevoeging geen waarde